doc-src/vm.rst

```diff
@@ -182,7 +182,7 @@ Build and provision
 To build the VM and install Docker, Charliecloud, etc.::
 
   $ cd packaging/vagrant
-  $ CH_VERSION=v0.9.1 vagrant up
```

Comment:
Looks like the meaning of `CH_VERSION` needs to be updated a few lines below.

Reply:
Should we have a way for people to pull the head of the master branch?